python/tests/test_variable.py

```diff
@@ -140,6 +140,20 @@ def test_create_1D_vector_3_float64():
     assert var.unit == sc.units.m
 
 
+@pytest.mark.skip(reason="enable once datetime64 is available with pybind")
+def test_create_datetime64():
+    var = sc.Variable(dims=['x'],
+                      values=[120, 121],
```

Comment:
While we don't have support for array access yet, I think if you omit the `values=[ ...]` and pass a `shape=[2]` you should be able to create a variable with this dtype, based on the bindings you added already.